src/backend/distributed/Makefile

```diff
@@ -5,7 +5,7 @@ citus_top_builddir = ../../..
 
 MODULE_big = citus
 EXTENSION = citus
-EXTVERSIONS = 5.0 5.0-1 5.0-2
+EXTVERSIONS = 5.0 5.0-1 5.0-2  5.1-1
```

Comment:
There's an extra space here. Kill it for consistency? Or is it intentional (to separate the `5.1` series?). If the latter, what do you think about separating each release on a separate line (would make it easier to see what the "next" schema would be for a given, extant release).